Makefile

```diff
@@ -10,7 +10,7 @@ include ${ENV_FILE}
 export $(shell sed 's/=.*//' ${ENV_FILE})
 endif
 
-CONTAINER_BUILDER?=podman
+CONTAINER_BUILDER?=docker
```

Comment:
We use `podman` as our default container builder due to license limitation of the docker engine nowadays. We would want to keep it that way in our automations, so we encourage overriding with docker in the local environment filest to the people that need it